src/stories/genreSelect.stories.ts

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import GenreSelect from '../components/genreSelect/GenreSelect';
+import { fireEvent, userEvent, within } from '@storybook/testing-library';
+
+const meta: Meta<typeof GenreSelect> = {
+  title: 'GenreSelect',
+  component: GenreSelect,
+};
+export default meta;
+type Story = StoryObj<typeof meta>;
+const genreList: Genre[] = ['Horror', 'Romantic', 'Thrilling'];
+
+export const EmptyGenreList: Story = {
+  args: {
+    genreList: [],
+    selectedGenre: undefined,
+    onSelect: (genre) => console.log('Genre Selected ' + genre),
+  },
+};
+
+export const NoDefaultGenreSelected: Story = {
```

Comment:
This is not a case. You need to define default in component default value for selected. Anyway (if selected provided or not) you need to csdhow some content to the user

Reply:
It defenetely should be 'All'
![Screenshot 2024-03-10 212424](https://github.com/weam1995/ReactMentoringProject/assets/79011727/b796eedc-d3f2-42ef-887e-241ab24806d7)